README.md

```diff
@@ -58,6 +58,14 @@ Run the production server from the bundled environment:
 
     bin/env bin/bundle exec bin/thin -p $PORT start
 
+### Deploy with Docker
+To deploy PromDash with Docker, use the [prom/promdash](https://registry.hub.docker.com/u/prom/promdash/).
+By default, the image will start thin in producation mode. To run
```

Comment:
Hehe, I was like, "What's a thin image?". Then I remembered that it's a webserver. But non-Rubyists could be confused. Maybe say, `the image will use the [thin webserver](http://code.macournoyer.com/thin/) in production mode.`